modules/hitlnext/src/views/full/app/storage.ts

```diff
@@ -0,0 +1,20 @@
+import serialize from 'serialize-javascript'
+
+import { MODULE_NAME } from '../../../constants'
+
+const deserialize = (serializedJavascript: string) => {
+  return eval('(' + serializedJavascript + ')')
```

Comment:
Is using eval absolutely necessary ?

Reply:
Well, nothing is absolutely necessary, but this is the simplest & fastest & smallest way I've found to deep serialize an object to localstorage. There are a lot of libs to serialize data but I've chosen https://www.npmjs.com/package/serialize-javascript for it's robustness and popularity. Happy to use another if you have a recommendation.